setup_default_backend.sh

```diff
@@ -31,6 +31,7 @@ function install_backend_system_dep {
     log "install deps for dealing with most preview..."
     $SUDO apt install -y zlib1g-dev libjpeg-dev && loggood "success" || logerror "some error"
     $SUDO apt install -y imagemagick libmagickwand-dev ghostscript libfile-mimeinfo-perl poppler-utils libimage-exiftool-perl && loggood "success" || logerror "some error"
+    $SUDO apt install -y libldap2-dev libsasl2-dev && loggood "success" || logerror "some error" # for ldap support
```

Comment:
What is the point of `&& loggood "success" || logerror "some error"` ?

Reply:
@buxx if install of package success or failed you can see this in log for this stage.